src/cloudkit.cs

```diff
@@ -11,23 +11,166 @@ namespace XamCore.CloudKit {
 	[iOS (8,0), Mac (10,10, onlyOn64 : true)]
 	[DisableDefaultCtor] // NSInvalidArgumentException Reason: You must call -[CKAsset initWithFileURL:] or -[CKAsset initWithData:]
 	[BaseType (typeof (NSObject))]
-	interface CKAsset : NSCoding, NSSecureCoding {
+	interface CKAsset : NSCoding, NSSecureCoding
+	{
 
 		[Export ("initWithFileURL:")]
 		IntPtr Constructor (NSUrl fileUrl);
 
 		[Export ("fileURL", ArgumentSemantic.Copy)]
 		NSUrl FileUrl { get; }
 	}
+	
+	[iOS (10,0), Watch (3,0), TV (10,0), Mac (10,12, onlyOn64 : true)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface CKUserIdentityLookupInfo : NSSecureCoding, NSCopying
+	{
+		[Static, Internal, Export ("initWithEmailAddress:")]
+		IntPtr _FromEmail (string emailAddress);
+
+		[Static, Internal, Export ("initWithPhoneNumber:")]
+		IntPtr _FromPhoneNumber (string phoneNumber);
+
+		[Static, Internal, Export ("initWithUserRecordID:")]
+		IntPtr _FromRecord (CKRecordID userRecordID);
+
+		[Static]
+		[Export ("lookupInfosWithEmails:")]
+		CKUserIdentityLookupInfo[] GetWithEmails (string[] emails);
```

Comment:
bad name, it does not tell what you `Get` but what's used, which is information that the parameter already provide). Use `GetLookupInfos`, it's closer to the selector.